src/libraries/System.Resources.Extensions/tests/System.Resources.Extensions.Tests.csproj

```diff
@@ -16,7 +16,15 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="System.Drawing.Common.TestData" Version="$(SystemDrawingCommonTestDataVersion)" />    
+    <ProjectReference Include="..\src\System.Resources.Extensions.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework.StartsWith('$(NetCoreAppCurrent)'))">
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Drawing.Common\src\System.Drawing.Common.csproj" />
```

Comment:
You'll also need System.Windows.Extensions to get the drawing TypeConverters.

Can you also double-check that you can run the GenerateTestResourcesFile target below and it doesn't change the .resources file?

Reply:
Thanks, I wasn't able to figure that out :(

> Can you also double-check that you can run the GenerateTestResourcesFile target below and it doesn't change the .resources file?

Already did yesterday. The target doesn't work in it's current state because `TargetDir` and `TargetFile` aren't set but I hardcoded the paths and the file didn't change.